src/components/Calculator.js

```diff
@@ -1,28 +1,20 @@
-import React, { Component } from 'react';
+import { useState } from 'react';
 import Display from './Display';
 import Keyboard from './Keyboard';
 
-class Calculator extends Component { /* eslint-disable-line react/prefer-stateless-function */
-  constructor(props) {
-    super(props);
-    this.updateDisplay = this.updateDisplay.bind(this);
-    this.state = {
-      displayOutput: '0',
-    };
-  }
+const Calculator = () => {
+  const [displayOutput, setDisplayOutput] = useState('0');
 
-  updateDisplay(newText) {
-    this.setState({ displayOutput: newText });
-  }
+  const updateDisplay = (newDisplayOutput) => {
+    setDisplayOutput(newDisplayOutput);
+  };
 
-  render() {
-    return (
-      <div className="Calculator">
-        <Display value={this.state.displayOutput} /> { /* eslint-disable-line */ }
-        <Keyboard onKeyPress={this.updateDisplay} />
-      </div>
-    );
-  }
-}
+  return (
+    <div className="Calculator">
+      <Display value={displayOutput} />
+      <Keyboard onKeyPress={updateDisplay} />
+    </div>
+  );
+};
 
 export default Calculator;
```

Comment:
- [ ] Please work on the `%` functionality currently its not working

Reply:
What exactly is "not working"? - I just tested it and it works fine. Please be more specific or provide a screenshot.

Reply:
### Hello @crgc,

We had a confusion about if we can check `%` as a percentage or `modulus
before I was checking percentage but got clarifications that it is modulus for this
project! I apologize for the inconvenience.

Lydia